src/platform/backends/qemu/iptables_config.cpp

```diff
@@ -96,6 +97,18 @@ void insert_iptables_rule(const QString& table, const QString& chain, const QStr
             fmt::format("Failed to set iptables rule for table {}: {}", table, process->read_all_standard_error()));
 }
 
+void append_iptables_rule(const QString& table, const QString& chain, const QStringList& rule)
+{
+    auto process = mp::ProcessFactory::instance().create_process(
+        iptables, QStringList() << wait << dash_t << table << append_rule << chain << rule);
+
+    auto exit_state = process->execute();
+
+    if (!exit_state.completed_successfully())
+        throw std::runtime_error(
+            fmt::format("Failed to set iptables rule for table {}: {}", table, process->read_all_standard_error()));
+}
```

Comment:
This duplication is unnecessary really. Here is [a patch avoiding it](https://paste.ubuntu.com/p/54NgXWrwq4/)

Reply:
Thanks, yes, we can avoid the duplication.  I was just trying to get a fix in quickly, but since it's not merged yet, then I'll just apply your patch:)